sns/cm/st/repair_test.sh

```diff
@@ -30,13 +30,11 @@ set -e
 # 3) Once i/o is done, start repair.
 # Note: This script should be run from motr directory
 
-M0_SRC_DIR=`readlink -f $0`
+M0_SRC_DIR=$("readlink -f $0")
```

Comment:
Have you tested this change? Does it work?

Reply:
There is no ST available, which consumes this file